### ISSUE TITLE
TextArea style bug on Chrome

### ISSUE THREAD

**Issue by ajdiyassin** [radix-ui/themes#167]
I have an issue where the TextArea outline is shown on the first render then when I click inside the outline disappears.

First render
![image](https://github.com/radix-ui/themes/assets/103142364/6801a554-3ae5-4fcb-a24f-82de9ff67637)

focus
![image](https://github.com/radix-ui/themes/assets/103142364/2cb03403-f3ee-41f6-842b-ea9765612525)

clicking outside 
![image](https://github.com/radix-ui/themes/assets/103142364/b4368ea1-8e4b-448c-874a-24c45b5cd59e)

**Comment by MrDogeBro** [comment]
I also seem to have this issue on Firefox. On Safari though it seems to be fine.

**Comment by o2dependent** [comment]
There seems to be an issue with extensions injecting markup and breaking the "+" adjacent sibling selector (Grammarly is what broke it for me). Changing the adjacent sibling selector ("+") to the general sibling selector ("\~") seems to fix that issue. I'm unsure if there is a reason for not using "~" though.

If you need this fixed ASAP try brute force overwriting the specificity in your styles and make a temp note as a CSS var to avoid headaches later when debugging if you think you'll forget like me.  If I have time I'll make a PR for this fix and check for other issues like this.

<details>
<summary><h2>Temp CSS overwrite example</h2></summary>
<p>This is for the surface variant with gray color. Change the variants/colors you use.</p>

```css
/* FIXME: Dangerously overwrite Radix-UI TextAreaInput styles */
:is(.rt-TextAreaRoot:where(.rt-variant-surface) :where(.rt-TextAreaInput)) ~ :where(.rt-TextAreaChrome) {
	--TEMP-FIX: "dangerously overwriting Radix-UI styles. check for updates and remove this if fixed.";
	box-shadow: inset 0 0 0 1px var(--gray-a7);
	background-color: var(--color-surface);
	padding: 1px;
	background-clip: content-box;
}
``` 
</details> 
<details>
<summary><h2>Here are some pictures with quick notes to show what's happening.</h2></summary>

![radix-textarea-bug__before-focus](https://github.com/radix-ui/themes/assets/46045126/304757d4-5e86-4d7e-973a-7366e4f68a7a)

![radix-textarea-bug__after-focus__bug](https://github.com/radix-ui/themes/assets/46045126/a027df66-dad8-48bf-ab3a-6e12d311086e)

![radix-textarea-bug__after-focus__fixed](https://github.com/radix-ui/themes/assets/46045126/9b6a90df-3e59-4b14-a5cf-ce6d87843a8b)

</details>

**Comment by vladmoroz** [comment]
> There seems to be an issue with extensions injecting markup and breaking the "+" adjacent sibling selector (Grammarly is what broke it for me). Changing the adjacent sibling selector ("+") to the general sibling selector ("~") seems to fix that issue. I'm unsure if there is a reason for not using "~" though.

This is really helpful, thanks. We’ll fix it.

**Comment by vladmoroz** [comment]
`TextArea` being hijacked by Grammarly is now fixed in a preview release, `2.1.0-rc.5`
